examples/bert/bert_cuda.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Suggest renaming the title of this config file to: bert_cuda_gpu.json. Same to the trt config file.

Reply:
Updated.